src/drone.ts

```diff
@@ -0,0 +1,30 @@
+const DRONE_API = "https://drone.gitea.io/api/repos/go-gitea/gitea";
+const HEADERS = {
+  Authorization: `Bearer ${Deno.env.get("BACKPORTER_DRONE_TOKEN")}`,
+};
+
+export const deleteBuild = async (buildNumber: number): Promise<void> => {
+  const response = await fetch(`${DRONE_API}/builds/${buildNumber}`, {
+    method: "DELETE",
+    headers: HEADERS,
+  });
+
+  if (!response.ok) {
+    throw new Error(
+      `Failed to delete build ${buildNumber}: ${response.statusText}`,
```

Comment:
While the API is DELETE, I think `cancel` or `stop` would be a more suitable term as the build still stays in the list of builds and it not delete. So I suggest rename the function and the message here.

Reply:
Done